packages/material-ui/src/Select/SelectInput.d.ts

```diff
@@ -3,6 +3,15 @@ import { SxProps } from '@material-ui/system';
 import { Theme } from '..';
 import { MenuProps } from '../Menu';
 
+/**
+ * The change can be caused by different kind of events.
+ * The type of event depends on what caused the change.
+ * For example, when the browser auto-fills the `Select` you'll receive a `React.ChangeEvent`.
+ */
+export type SelectChangeEvent<T = string> =
```

Comment:
`string` is the most common use case. IMO people should not use any other type to begin with. There was never a reason to loosen the type. Input types on the web should just be strings. Everything else just makes things more difficult than they need to be.